pkg/controller/controller_binding_test.go

```diff
@@ -413,6 +418,23 @@ func TestReconcileServiceBindingWithParameters(t *testing.T) {
 	}
 	binding.Spec.Parameters = &runtime.RawExtension{Raw: b}
 
+	if err := testController.reconcileServiceBinding(binding); err != nil {
+		t.Fatalf("unexpected error: %v", err)
+	}
+
+	expectedParameters := map[string]interface{}{
+		"args": []interface{}{
+			"first-arg",
+			"second-arg",
+		},
+		"name": "test-param",
+	}
+	expectedParametersChecksum := generateChecksumOfParametersOrFail(t, expectedParameters)
+
+	binding = assertServiceBindingOperationInProgressWithParametersIsTheOnlyCatalogAction(t, fakeCatalogClient, binding, v1beta1.ServiceBindingOperationBind, expectedParameters, expectedParametersChecksum)
+	fakeCatalogClient.ClearActions()
+	fakeKubeClient.ClearActions()
```

Comment:
You clear `fakeKubeClient` actions without checking them (here and in other places). My thinking is:
1. If you don't care about fakeKubeClient actions - don't clear them
2. If you do care then you should assert them before clearing?

Or is it just for making sure that if we ever need to check them in the future, they won't be confusing?

Reply:
Yeah, you're right. The tests should check the actions on all three clients after each call to reconcile(). I'll add the checks.